src/test/functional/pages/BuzzPage.ts

```diff
@@ -1,20 +1,30 @@
 import { BasePage } from "./BasePage";
+
 export class BuzzPage extends BasePage {
     protected readonly sharePhotosButton = this.page.getByRole("button", { name: "Share Photos" })
     protected readonly fileInput = this.page.locator('input[type="file"]');
     protected readonly submitPhoto = this.page.getByRole("button", { name: 'Share', exact: true })
-    protected readonly threeDotsIcon = this.page.locator('i.oxd-icon.bi-three-dots');
+    protected readonly threeDotsIcon = this.page.getByRole('button', { name: '' });
     protected readonly deletePostParagraph = this.page.getByText("Delete Post"); 
     protected readonly editPostParagraph = this.page.getByText("Edit Post");
     protected readonly confirmDeleteButton = this.page.getByRole("button", { name: "Yes, Delete" })
-    public readonly photoImg = this.page.locator('.orangehrm-buzz-photos-item img')
-    public readonly buzzPageButton = this.page.locator('span:has-text("Buzz")')
-    protected readonly textFieldInPostEdit = this.page.locator('.orangehrm-buzz-post-modal-header-text .oxd-buzz-post .oxd-buzz-post-input')
-    protected readonly removePictureButton = this.page.locator('.oxd-icon-button.orangehrm-photo-input-remove i.bi-x')
-    protected readonly confirmEditedPost = this.page.locator('.orangehrm-buzz-post-modal-actions button')
-    
-    async sharePost(filePath:string): Promise<void> {
+    public readonly buzzPageButton = this.page.getByRole('link', { name: 'Buzz' })
+    protected readonly textFieldInPostEdit = this.page.getByRole('dialog').locator('textarea')
+    protected readonly confirmEditedPost = this.page.getByRole('dialog').getByRole('button', { name: 'Post' })
+    protected readonly shareVideoButton = this.page.getByRole('button', { name: 'Share Video' })
+    protected readonly pasteUrlTextarea = this.page.getByPlaceholder("Paste Video URL")
```

Comment:
I think you should decide if you use single quotes or double quotes. :) You have a mix of both in those locators.

Reply:
dislexia?